tools/wallet/src/key_gen.rs

```diff
@@ -21,6 +20,16 @@ pub fn keygen() -> (AuthenticationKey, AccountAddress, WalletLibrary, String) {
     (auth_key, account, wallet, mnemonic_string)
 }
 
+/// Runs the key generation process asynchronously.
```

Comment:
I do like this pattern of being explicit with the parameter documentation.
Though in general we don't see that pattern in rust.
This is a more idiomatic way of writing comments: https://www.tangramvision.com/blog/making-great-docs-with-rustdoc

Reply:
Updated doc.